storage/aptosdb/src/test_helper.rs

```diff
@@ -109,20 +108,16 @@ pub fn update_in_memory_state(state: &mut StateDelta, txns_to_commit: &[Transact
         if txn_to_commit.is_state_checkpoint() {
             state.current = state
                 .current
-                .clone()
-                .freeze()
```

Comment:
why is this freeze and unfreeze removed?

Reply:
just a different way to hack.. (SparseMerkleTree::batch_update() now generates `Usage::Untracked` internally, as is the intention here)